green/test/test_windows.py

```diff
@@ -23,3 +23,6 @@ def test_colorOutput(self):
         self.assertTrue(issubclass(type(gs.stream),
                         colorama.ansitowin32.StreamWrapper))
 
+    def test_unicode_override(self):
+        pass
```

Comment:
It is a poor, sad test :(

I couldn't figure out how to get `green` to run a `unittest` testcase, and then validate what gets printed out. The [actual testcase](https://github.com/CleanCut/green/issues/119#issuecomment-209138184) is nearly trivial...

```
class UnicodeTest(unittest.TestCase):
    def test1(self):
        print('\u03BB')  # lambda
        self.assertTrue(True)
```

Suggestions on how to go about this?